src/pages/new-item/new-item.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from '@angular/core';
+import {IonicPage, NavController} from 'ionic-angular';
+import {AngularFireDatabase, AngularFireList} from "angularfire2/database";
+
+/**
+ * Generated class for the NewItemPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-new-item',
+  templateUrl: 'new-item.html',
+})
+export class NewItemPage {
+
+  private _itemRef: AngularFireList<any>;
+
+  private itemName: string = "";
+  private itemDescription: string = "";
+  private lendTime: number = 7;
+  private itemImgUrl: string = "";
+  private credit: number = 10;
+
+  constructor(private db:AngularFireDatabase, private navCtrl: NavController) {
+    this._itemRef = this.db.list('/item');
+  }
+
+  addItem(){
+    this._itemRef.push({
+      credit: this.credit,
+      image_url: this.itemImgUrl,
+      name: this.itemName,
+      rating: 0,
+      reviews_num: 0,
+      status: "on_shell",
```

Comment:
We should potentially store all possible statuses as constants somewhere so we don't have to worry about typos